src/sentry/api/endpoints/team_projects.py

```diff
@@ -214,4 +217,16 @@ def post(self, request: Request, team) -> Response:
                 sender=self,
             )
 
+            # Create project option to turn on ML similarity feature for new EA projects
+            seer_eligible_platform = (
+                project.platform and project.platform in SEER_ELIGIBLE_PLATFORMS
+            )
+            if (
+                project
```

Comment:
I think this should already be truthy by this point since line 222 is looking at `project.platform`?